src/components/Navbar.js

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+import '../styles/nav.css';
 
 function Navigation() {
   return (
-    <div>
+    <nav>
       <ul>
         <li>
-          <Link to="/">Books</Link>
+          <h1>Bookstore CMS</h1>
```

Comment:
[OPTIONAL] You did a great job with the design. But fonts does not match with the Zeplin design. Kindly fix them.
<img width="460" alt="Screenshot 2023-03-04 at 8 13 37 PM" src="https://user-images.githubusercontent.com/25887253/222919707-3f026c4d-c8e3-4e90-99c4-14831b274e90.png">
<img width="1025" alt="Screenshot 2023-03-04 at 8 14 06 PM" src="https://user-images.githubusercontent.com/25887253/222919710-83e0f6da-4c5c-4ce6-859d-3cd52ab66ce6.png">